manifest.json

```diff
@@ -26,7 +26,7 @@
         "mysql"
     ],
     "requirements": {
-        "yunohost": ">= 4.2.0"
+        "yunohost": ">= 4.3.0"
```

Comment:
This change doesn't strictly require Yunohost 4.3, does it? It doesn't seem to introduce new functions, only use another path if some env var is defined.

Could it be made compatible with previous versions of Yunohost? This would avoid to let users running old versions of Mattermost, just because they didn't upgrade their Yunohost version.

Reply:
yes it does need 4.3 ...

Reply:
From what I understand, the "purge" option works with an env var. So if we simply drop the version requirement, on previous Yunohost versions, it will still work: the `YNH_APP_PURGE` env var will not be defined, default to 0, and the files will (correctly) not be purged. And on Yunohost >= 4.3, the env var will be populated with the correct value.

What do you think about removing the required version bump and then merging this?